2022-11-13 14:52:22.418 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:53:22.640 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:53:24.447 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:53:24.488 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:53:24.490 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:53:38.751 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:53:38.793 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:53:38.795 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:53:54.631 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:53:57.421 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:53:57.461 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:53:57.463 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:54:19.758 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:54:21.630 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:54:21.670 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:54:21.671 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:54:44.591 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:54:45.391 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:54:45.431 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:54:45.433 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:55:46.975 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:55:48.744 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:55:48.784 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:55:48.786 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:56:34.453 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:56:37.833 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:56:37.873 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:56:37.874 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:57:06.646 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:57:08.702 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:57:08.742 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:57:08.744 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:57:18.756 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:57:48.965 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:57:49.005 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:57:49.007 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:58:05.070 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 14:58:16.179 | INFO     | __main__:main:31 - Starting bot
2022-11-13 14:58:16.220 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 14:58:16.222 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 14:59:32.149 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 20:01:46.798 | INFO     | __main__:main:31 - Starting bot
2022-11-13 20:25:32.065 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 20:26:19.154 | INFO     | __main__:main:31 - Starting bot
2022-11-13 20:26:19.169 | ERROR    | database.database:create_table:44 - Exception create_table:  table wallets already exists
2022-11-13 20:26:19.170 | ERROR    | database.database:add_wallets:150 - Exception add_wallets:  UNIQUE constraint failed: wallets.id
2022-11-13 20:27:49.135 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-13 15:32:45.534 | INFO     | __main__:main:31 - Starting bot
2022-11-13 15:34:05.822 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-14 20:55:15.140 | INFO     | __main__:main:31 - Starting bot
2022-11-14 21:09:26.688 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-14 21:10:21.853 | INFO     | __main__:main:31 - Starting bot
2022-11-14 21:16:29.253 | ERROR    | __main__:<module>:61 - Bot stopped!
2022-11-22 12:53:37.460 | INFO     | __main__:main:31 - Starting bot
2022-11-22 14:10:34.581 | ERROR    | __main__:<module>:61 - Bot stopped!
